StardewArchipelago/Archipelago/Gifting/GiftGenerator.cs

```diff
@@ -160,6 +160,8 @@ private IEnumerable<GiftTrait> GetCategoryTraits(ObjectData objectInfo)
                 }
             }
 
+            var type = objectInfo.Type ?? string.Empty;

```

Comment:
This doesn't work **in addition** to line 147, it needs to be **instead**. As iut currently is, I'm pretty sure it won't even build.